helix-core/src/main/java/org/apache/helix/HelixCloudProperty.java

```diff
@@ -58,8 +59,13 @@ public class HelixCloudProperty {
   private List<String> _cloudInfoSources;
 
   // The name of the function that will fetch and parse cloud instance information.
+  @Deprecated
```

Comment:
For backward compatible, may be we can let 

_cloudInfoProcessorFullyQualifiedClassName = pkg name + _cloudInfoProcessorName.

Up to you. The reason is that it should be reflected to some functions as validator which does not exists today. Maybe we need a todo here.

Reply:
I decided to make a new attribute called `_cloudInfoProcessorPackage` and make `getCloudInfoProcessorFullyQualifiedClassName` return `_cloudInfoProcessorPackage + "." + _cloudInfoProcessorName`